src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
fix Random
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/bc47f22a-6058-4ae4-a91c-c6a4393973c5)

Reply:
You should fix (or answer to) all mentor's comments before re-request new review. Be attentive and do not hesitate to contact us in Slack if you do not understand something.
Make Random a class-level variable.